impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -481,60 +473,62 @@ private String deriveLoginModuleFromSasl(String saslMechanism) {
         }
     }
 
-    private static void configureInternalListener(TreeMap<String, String> protocolMap, TreeMap<String, String> listeners, KafkaEndpoints.EndpointPair interBrokerEndpoint,
+    private static void configureInternalListener(TreeMap<String, String> protocolMap, TreeMap<String, String> listeners, KafkaListener interBrokerEndpoint,
                                                   TreeMap<String, String> advertisedListeners, TreeSet<String> earlyStart, Properties server) {
         protocolMap.put(INTERNAL_LISTENER_NAME, SecurityProtocol.PLAINTEXT.name());
-        listeners.put(INTERNAL_LISTENER_NAME, interBrokerEndpoint.listenAddress());
-        advertisedListeners.put(INTERNAL_LISTENER_NAME, interBrokerEndpoint.advertisedAddress());
+        listeners.put(INTERNAL_LISTENER_NAME, interBrokerEndpoint.bind().toString());
```

Comment:
nit: I don't mind the change from `listenAddress` to `bind` but needing to `toString` here feels like a step backwards.